contracts/ConvictionVoting.sol

```diff
@@ -239,8 +247,9 @@ contract ConvictionVoting is AragonApp, TokenManagerHook {
         require(proposal.proposalStatus == ProposalStatus.Active, ERROR_PROPOSAL_NOT_ACTIVE);
 
         proposal.proposalStatus = ProposalStatus.Cancelled;
+        _closeDisputableAction(proposal.agreementActionId);
```

Comment:
is it ok to close the disputableAction before the `vault.transfer` was effectively done?

Reply:
Yeah that should be fine. In fact we typically want to do transfers, of any type, after state changes to reduce the risk of reentrancy attacks.